CHANGELOG.rst

```diff
@@ -20,17 +20,23 @@ Unreleased
 [0.2.0] - 2021-10-04
 ********************
 
+.. _added_0.2.0:
```

Comment:
@pshiu: Separately, if you temporarily update this to `.. _added 0.2.0:` or `.. _anything`, do you get a different result?